Vikta-UI-testing-sample/src/test/java/com/griddynamics/qa/vikta/uitesting/sample/stepsDefinitions/BaseSteps.java

```diff
@@ -50,6 +50,12 @@ TestDataAndProperties getData() {
     return PageFactory.initElements(getDriver(), pageClass);
   }
 
+  @Step
+  void scrollToFooter(){
+    ((JavascriptExecutor) driver)
```

Comment:
have You tried selenium Actions to do the scrolling? For example:

WebElement element = driver.findElement(By.id("my-id"));
Actions actions = new Actions(driver);
actions.moveToElement(element);
actions.perform();